ciao-scheduler/scheduler.go

```diff
@@ -477,12 +490,35 @@ func (sched *ssntpSchedulerServer) getWorkloadResources(work *payloads.Start) (w
 	return workload, nil
 }
 
+func networkDemandsSatisfied(node *nodeStat, workload *workResources) bool {
+	if node.isNetNode {
+		var matchedNetworksCount int
+		var requestedNetworksCount int
+
+		for _, requestedNetwork := range workload.physNets {
+			requestedNetworksCount++
+			for _, availableNetwork := range node.networks {
+				if requestedNetwork == availableNetwork.NodeIP {
```

Comment:
@tpepper we may want to provide a generic satisfies method for non integer matches. As the comparisons may get complicated in the future. Here for example a subnet that is a subnet of the demanded subnet may also be ok.